### PR TITLE
Case sensitive check and Jaro distance

### PR DESCRIPTION
**Pull request by Safranil**
Added the option to enable and disable the case sensitive checks and added the computing of Jaro distance.

I also fixed a typo in the English lang file (line 112).

### CODE REVIEW

**Comment by chdemko** [comment]
You have a lot of php style failing see https://travis-ci.org/e-lang/moodle-mod_elang/builds/132072565

**Comment by chdemko** [comment]
You have a lot of PHP style failing. See https://travis-ci.org/e-lang/moodle-mod_elang/builds/132072565

**Comment by Safranil** [comment]
Indeed, I checked all error. The build is now successful.

File: src/server/server.php [hunk 1]
```diff
@@ -339,14 +339,34 @@
...
+		}
+		else
+		{
+			$parsed_text = $text;
+			$parsed_content = $elements[$number]['content'];
+		}
+
+		$previous_locale = setlocale(LC_ALL, 0);
+
+		if (false === setlocale(LC_ALL, str_replace("-",  "_", $options["language"])))
```
Comment (chdemko) [inline src/server/server.php]: It seems that windows uses different strategy for naming languages see
http://php.net/manual/fr/function.setlocale.php
http://stackoverflow.com/questions/9489339/setlocalelc-all-en-gb-utf8-not-working-on-windows
https://msdn.microsoft.com/en-us/library/39cwe7zf(v=vs.90).aspx

Comment (chdemko) [inline src/server/server.php]: For linux, a way is to test different value for the locale:
~~~
setlocale(LC_ALL, 'fr_FR.UTF8', 'fr.UTF8', 'fr_FR.UTF-8', 'fr.UTF-8')
~~~
See
http://php.net/manual/fr/function.setlocale.php#87038

**Pull request merged**